Model/ConfigProvider/Method/PayByBank.php

```diff
@@ -0,0 +1,212 @@
+        [
+            'name' => 'N26',
+            'code' => 'NTSBDEB1',
+            'imgName' => 'n26'
+        ]
+    ];
+    /**
+     * @var CustomerSession
+     */
+    protected $customerSession;
+
+    /**
+     * @param Repository           $assetRepo
+     * @param ScopeConfigInterface $scopeConfig
+     * @param AllowedCurrencies    $allowedCurrencies
+     * @param PaymentFee           $paymentFeeHelper
+     * @param FormKey              $formKey
+     */
+    public function __construct(
+        Repository $assetRepo,
+        ScopeConfigInterface $scopeConfig,
+        AllowedCurrencies $allowedCurrencies,
+        PaymentFee $paymentFeeHelper,
+        CustomerSession $customerSession
+    ) {
+        parent::__construct($assetRepo, $scopeConfig, $allowedCurrencies, $paymentFeeHelper);
+        $this->customerSession = $customerSession;
+    }
+
+    /**
+     * @var array
+     */
+    protected $allowedCurrencies = [
+        'EUR'
+    ];
+
+    /**
+     * @return array|void
+     */
+    public function getConfig()
+    {
+        if (!$this->scopeConfig->getValue(
+            static::XPATH_PAYBYBANK_ACTIVE,
+            \Magento\Store\Model\ScopeInterface::SCOPE_STORE
+        )) {
+            return [];
+        }
+
+
+        $selectionType = $this->scopeConfig->getValue(
+            self::XPATH_PAYBYBANK_SELECTION_TYPE,
+            \Magento\Store\Model\ScopeInterface::SCOPE_STORE
+        );
+
+        return [
+            'payment' => [
+                'buckaroo' => [
+                    'paybybank' => [
+                        'banks' => $this->getIssuersWithSelected(),
+                        'subtext'   => $this->getSubtext(),
+                        'subtext_style'   => $this->getSubtextStyle(),
+                        'subtext_color'   => $this->getSubtextColor(),
+                        'allowedCurrencies' => $this->getAllowedCurrencies(),
+                        'selectionType' => $selectionType,
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @param null|int $storeId
+     *
+     * @return float
+     */
+    public function getPaymentFee($storeId = null)
+    {
+        return 0;
+    }
+
+    /**
+     * @param null|int $storeId
+     *
+     * @return string
+     */
+    public function getPaymentFeeLabel($storeId = null)
+    {
+        return  $this->scopeConfig->getValue(
+            Account::XPATH_ACCOUNT_PAYMENT_FEE_LABEL,
+            \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
+            $storeId
+        );
+    }
+
+    /**
+     * Get bank issuers, move last saved bank issuer first,
+     * add selected property to the issuers array
+     *
+     * @return array
+     */
+    public function getIssuersWithSelected()
+    {
+        $issuers = $this->formatIssuers();
+        $customer = $this->customerSession->getCustomer();
+        $savedBankIssuer = $customer->getData(PayByBankMethod::EAV_LAST_USED_ISSUER_ID);
```

Comment:
Where is set: PayByBankMethod::EAV_LAST_USED_ISSUER_ID, buckaroo_last_paybybank_issuer?